src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -72,6 +81,32 @@ export default class PatientViewMutationTable extends MutationTable<IPatientView
         // patient view has a custom renderer for protein change column, so we need to override render function
         this._columns[MutationTableColumnType.PROTEIN_CHANGE].render = ProteinChangeColumnFormatter.renderFunction;
 
+        // customization for allele count columns
+
+        this._columns[MutationTableColumnType.REF_READS_N] = {
+            ...this._columns[MutationTableColumnType.REF_READS_N],
+            render: (d:Mutation[])=>AlleleCountColumnFormatter.renderFunction(d, this.getSamples(), "normalRefCount"),
+            download: (d:Mutation[])=>AlleleCountColumnFormatter.getTextValue(d, this.getSamples(), "normalRefCount")
+        };
+
+        this._columns[MutationTableColumnType.VAR_READS_N] = {
+            ...this._columns[MutationTableColumnType.VAR_READS_N],
+            render: (d:Mutation[])=>AlleleCountColumnFormatter.renderFunction(d, this.getSamples(), "normalAltCount"),
+            download: (d:Mutation[])=>AlleleCountColumnFormatter.getTextValue(d, this.getSamples(), "normalAltCount")
+        };
+
+        this._columns[MutationTableColumnType.REF_READS] = {
+            ...this._columns[MutationTableColumnType.REF_READS],
+            render: (d:Mutation[])=>AlleleCountColumnFormatter.renderFunction(d, this.getSamples(), "tumorRefCount"),
+            download: (d:Mutation[])=>AlleleCountColumnFormatter.getTextValue(d, this.getSamples(), "tumorRefCount")
+        };
+
+        this._columns[MutationTableColumnType.VAR_READS] = {
```

Comment:
this is slightly confusing to read, plz consider changing to 
`this.columns[..].render = ` and `this.columns[..].download = `